src/coreclr/dir.common.props

```diff
@@ -14,6 +14,8 @@
     <TargetArchitecture Condition="'$(TargetArchitecture)' == ''">$(__BuildArch)</TargetArchitecture>
     <Configuration Condition="'$(Configuration)'==''">$(__BuildType)</Configuration>
 
+    <TargetArchitecture Condition="'$(TargetArchitecture)'==''">x64</TargetArchitecture>
```

Comment:
Would it make sense to move it two lines higher?

Also, why is not __BuildArch set? Unrelated, but it would explain what I see in #34254.

Reply:
This is minimal change to get to a good state again before doing more changes. I am trying to maintain the order from https://github.com/dotnet/runtime/pull/34261/files#diff-7413e291b39b6f45a27f12bd3056c5daL15 .

Ultimately, this should be all deleted and reconciled with the libraries defaults  https://github.com/dotnet/runtime/blob/master/src/libraries/Directory.Build.props#L37

> it would explain what I see in #34254.

@safern 's change #34300 should fix this.

Reply:
I asked about the order only because it would group the `TargetArchitecture` fallback paths next to each other thus making it easier to read.

Thanks, I noticed @safern's change meanwhile.

Reply:
Yeah, I think it makes sense to group both `TargetArchitecture` declarations together as sequencial lines.